.travis.yml

```diff
@@ -23,3 +23,9 @@ jobs:
   #     - dapp testnet &
   #     - ./bin/deploy.sh
   #     - npm run test
+  - stage: Publish patch
+    if: "(NOT type IN (pull_request)) AND (branch = master)"
+    script:
+      - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
+      - npm run publish:patch
```

Comment:
Can we actually do the same here as we do on axis, i. e. publish canary versions only for now? 

I think we should keep npm as unstable until rev pools are done.

Reply:
I couldn't figure out how to do that with npm version. Lerna has built in functionality for this, but npm version doesn't appear to. I'll double check this.

Reply:
This is now fixed!